costmodel/aggregations.go

```diff
@@ -166,36 +186,68 @@ func mergeVectors(costDatum *CostData, aggregation *Aggregation, discount float6
 	}
 }
 
-func getPriceVectors(costDatum *CostData, discount float64, idleCoefficient float64) ([]*Vector, []*Vector, []*Vector, [][]*Vector) {
+func getPriceVectors(cp cloud.Provider, costDatum *CostData, discount float64, idleCoefficient float64) ([]*Vector, []*Vector, []*Vector, [][]*Vector) {
+	cpuCostStr := costDatum.NodeData.VCPUCost
+	ramCostStr := costDatum.NodeData.RAMCost
+	gpuCostStr := costDatum.NodeData.GPUCost
+	pvCostStr := costDatum.NodeData.StorageCost
+
+	// If custom pricing is enabled and can be retrieved, replace
+	// default cost values with custom values
+	customPricing, err := cp.GetConfig()
```

Comment:
Don't swallow the error here, since it's entirely possible we lost touch with the filesystem somehow. I'd say log it.

Reply:
Ok logging sounds good. Think it's ok to move forward like this, just skipping the custom pricing?

Reply:
I'd say yeah, I think it's okay to just use the cloud pricing if you can't fetch custom pricing. We can revisit though, but this should be pretty rare.